programs/drift/src/instructions/admin.rs

```diff
@@ -3439,6 +3442,31 @@ pub fn handle_delete_prelaunch_oracle(
     Ok(())
 }
 
+pub fn handle_initialize_pyth_pull_oracle(
+    ctx: Context<InitPriceFeed>,
```

Comment:
might make this context InitPythPullPriceFeed so its clear in the future if we need similar admin fn for switchboard or s.t.